miniupnpc/upnpc.c

```diff
@@ -705,15 +709,19 @@ int main(int argc, char ** argv)
 				break;
 			case 2:
 				printf("Found a (not connected?) IGD : %s\n", urls.controlURL);
+				if (!ignore) goto invalid;
 				printf("Trying to continue anyway\n");
 				break;
 			case 3:
 				printf("UPnP device found. Is it an IGD ? : %s\n", urls.controlURL);
+				if (!ignore) goto invalid;
 				printf("Trying to continue anyway\n");
 				break;
 			default:
 				printf("Found device (igd ?) : %s\n", urls.controlURL);
+				if (!ignore) goto invalid;
 				printf("Trying to continue anyway\n");
+				break;
```

Comment:
?

Reply:
Well, I added break for the last case of switch. But it is not needed.